bdd/spec/026_datastore_spec.sh

```diff
@@ -27,7 +27,7 @@ Describe "@std/datastore"
       on waitAndInc fn (ms: int64) {
         wait(ms);
         let i = namespace('foo').getOr('inc', 0);
-        i = i + 1;
+        i = i + 1 || 0;
```

Comment:
Incrementing is annoying with Result-wrapped math outputs. The saturating math operators in the RFC would probably go better here.